specification/monitor/resource-manager/readme.md

```diff
@@ -759,15 +759,15 @@ directive:
 
 ```
 
-``` yaml $(go) || $(csharp) || $(validation) || $(typescript)
+``` yaml ($(go) && !$(track2)) || $(csharp) || $(validation) || $(typescript)
 directive:
 - from: activityLogAlerts_API.json
   where: $.definitions
   transform: delete $["Resource"]
   reason: Missing kind, etag
 ```
 
-``` yaml !$(python) && !$(go) && !$(java) && $(tag) == 'package-2021-04'
+``` yaml !$(python) && (!$(go) && !$(track2)) && !$(java) && $(tag) == 'package-2021-04'
```

Comment:
This one previously appears already exclude for all golang (track1 or track2) by `!$(go)`?

Reply:
Good catch, I will remove this

Reply:
Fixed